src/DetailsView/components/failure-instance-panel-control.tsx

```diff
@@ -45,11 +49,24 @@ export class FailureInstancePanelControl extends React.Component<FailureInstance
 
     constructor(props) {
         super(props);
+        let filledOriginalInstance;
```

Comment:
I think "filledOriginalInstance" might not be such a great name for this; it might not be an instance handed to it.

I think current instance is fine.

Reply:
I was thinking "filledOriginalInstance" because the one handed to it may be null, in which case we fill it with empty (but valid) parameters. I get what you're saying though--I'll change it! Thanks!